### ISSUE TITLE
joint_limits values seem not realistic

### ISSUE THREAD

**Issue by phy25** [scottlx/Wheelchair-Arm-Control#2]
Hi, as far as I see values in `arm_moveit_config/config/joint_limits.yaml` is not in accordance with `my_arm/urdf/my_arm.urdf`:

- Acceleration limit is not set;
- Velocity limit's choice of 1 seems arbitary;
- `base2base_joint` has no limit at all:

https://github.com/scottlx/Wheelchair-Arm-Control/blob/96b62bff482b034f0aeb5c6884bbff984d55af50/arm_moveit_config/config/joint_limits.yaml#L5-L9

https://github.com/scottlx/Wheelchair-Arm-Control/blob/96b62bff482b034f0aeb5c6884bbff984d55af50/my_arm/urdf/my_arm.urdf#L163-L168

Acutally to my understanding `base2base_joint` is a rotatable panel. If it has motor attached, it should have velocity and joint limits on the (MoveIt!) controller level. If not, it shouldn't exist in `controller.yaml` at all, and its joint type can be set to `fixed`, maybe.

Just my two cents. There might be other considerations I am not aware here.

**Comment by scottlx** [comment]
Thanks for your review. Without your comment, I did not even notice this file. It is generated automatically bu moveit setup assistant. But as you can see in its annotation, th joint_limits.yaml is not used unless we change the config in the launch file to override our urdf joint properties. So the joint_limits.yaml is useless in our project.

As for the base2base_joint, I will add the velocity limit to it later. The reason I leave it blank here is that I have not interfaced the moveit with low level control. We have some problems like PID tune to deal with. 

I really appreciate your patient review. Some of the code is generated by packages in ROS. So I am really surprise that you read them. Thanks again for your comment!

**Comment by phy25** [comment]
You are right, `joint_limits.yaml` is in addition to URDF, not a replacement. It should be an important file MoveIt! relies on to set soft limits, so that's why I am looking at it. Didn't noticed the comment in that file, sorry 😔

And just out of curiosity, why do you choose velocity limit of "1"(m/s)? Are there any reference to the parameter?
